pages/api/id/[form]/owners.ts

```diff
@@ -38,5 +40,47 @@ export async function getEmailListByFormID(
   //Could not find formID in the path
   return res.status(400).json({ error: "Malformed API Request FormID not define" });
 }
-
-export default isRequestAllowed(["GET", "POST"], isUserSessionExist(owners));
+/**
+ * It will activate and deactivate all the owners associated to a specific form.
+ * This function is expecting to find a payload in the request body like:
+ * { "email": "forms@cds.ca", "active":"0/1"}
+ * "email": The email of the user associated to the form
+ * "active": Boolean value indicating whether the form owner is active or not
+ * @param req
+ * @param res
+ */
+export async function activateOrDeactivateFormOwners(
+  req: NextApiRequest,
+  res: NextApiResponse
+): Promise<void> {
+  //Extracting req body
+  const requestBody = req.body ? JSON.parse(req.body) : undefined;
+  //Payload validation
+  if (!requestBody?.email || !requestBody?.active) {
+    //Invalid payload
+    return res.status(400).json({ error: "Invalid payload fields are not define" });
+  } else {
+    const { email } = requestBody; // TODO email should also be checked i.e (valid Gov email). See #491
+    let { active } = requestBody;
```

Comment:
active should just be a boolean in JSON. JSON just supports true or false booleans